pkg/server/routes.go

```diff
@@ -15,17 +15,32 @@
 package server
 
 import (
+	"os"
+	"net/http"
+
 	"github.com/gobuffalo/packr/v2"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	log "github.com/sirupsen/logrus"
 )
 
+// GetAssetsFS returns http.FyleSystem for assets
+func (s *Server) GetAssetsFS() http.FileSystem {
```

Comment:
This will not work since `s.Routes()` is called before configuration setup (see: https://github.com/fydrah/loginapp/blob/master/cmd/serve.go#L45)

Assets will always be defaulted to embed assets

We must move `s.Routes()` call after configuration setup:
* Just after https://github.com/fydrah/loginapp/blob/master/pkg/server/server.go#L128
```
[...]
func (s *Server) Run() error {
	s.client = client.New(&s.Config.OIDC)
	s.Routes()  // <<<<<<
	if err := s.client.TLSSetup(); err != nil {
[...]
```
* remove route setup call in server `New()` func:
```
[...]
// New initialize a new server
func New(cfg *config.App) *Server {
	s := new(Server)
	s.Config = cfg
	s.router = httprouter.New()
//   <<<< call removed
	return s
}
```

Reply:
Fixed